.github/workflows/ci.yaml

```diff
@@ -107,6 +107,71 @@ jobs:
             # cache node modules using the same key as restore.
           key: ${{ steps.node_modules-cache-restore.outputs.cache-primary-key }}
 
+  E2E-Test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql
```

Comment:
Will it let you specify an image tag?

```
image: mysql:8
```

That will match what we have in our setup.  If you can't, it's fine.